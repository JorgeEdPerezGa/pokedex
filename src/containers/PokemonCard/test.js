import React from 'react';
import { shallow } from 'enzyme';
import { PokemonCard } from './index';

describe('PokemonCard', () => {
  const renderedComponent = shallow(<PokemonCard type={{}}/>);

  it('should match snapshot', async () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
