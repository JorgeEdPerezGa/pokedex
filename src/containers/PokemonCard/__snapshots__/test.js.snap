// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonCard should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PokemonCard
    type={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article
        className="pokemon-card"
        onClick={[Function]}
      >
        <p
          className="type-title"
        />
      </article>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p
          className="type-title"
        />,
        "className": "pokemon-card",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "type-title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      "type": "article",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article
          className="pokemon-card"
          onClick={[Function]}
        >
          <p
            className="type-title"
          />
        </article>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p
            className="type-title"
          />,
          "className": "pokemon-card",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "type-title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        "type": "article",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
